2015-01-20T06:26:40.553520+00:00 app[web.1]:   File "server.py", line 2, in <module>
2015-01-20T06:26:40.553528+00:00 app[web.1]:     from Ziggeo import Ziggeo
2015-01-20T06:26:40.553529+00:00 app[web.1]:   File "/app/Ziggeo.py", line 6, in <module>
2015-01-20T06:26:40.553551+00:00 app[web.1]:   File "/app/ZiggeoAuth.py", line 1, in <module>
2015-01-20T06:26:40.553579+00:00 app[web.1]:     from Crypto import Random
2015-01-20T06:26:40.553639+00:00 app[web.1]: ImportError: No module named Crypto
2015-01-20T06:26:40.553531+00:00 app[web.1]:     from ZiggeoAuth import ZiggeoAuth
2015-01-20T06:26:41.333576+00:00 heroku[web.1]: Process exited with status 1
2015-01-20T06:26:41.340320+00:00 heroku[web.1]: State changed from starting to crashed
2015-01-20T06:26:41.340320+00:00 heroku[web.1]: State changed from crashed to starting
2015-01-20T06:26:46.485706+00:00 heroku[web.1]: Starting process with command `python server.py`
2015-01-20T06:26:48.526858+00:00 app[web.1]: Traceback (most recent call last):
2015-01-20T06:26:48.526879+00:00 app[web.1]:     from Ziggeo import Ziggeo
2015-01-20T06:26:48.526873+00:00 app[web.1]:   File "server.py", line 2, in <module>
2015-01-20T06:26:48.526881+00:00 app[web.1]:   File "/app/Ziggeo.py", line 6, in <module>
2015-01-20T06:26:48.526909+00:00 app[web.1]:     from ZiggeoAuth import ZiggeoAuth
2015-01-20T06:26:48.526928+00:00 app[web.1]:   File "/app/ZiggeoAuth.py", line 1, in <module>
2015-01-20T06:26:48.526957+00:00 app[web.1]:     from Crypto import Random
2015-01-20T06:26:48.526977+00:00 app[web.1]: ImportError: No module named Crypto
2015-01-20T06:26:49.306657+00:00 heroku[web.1]: Process exited with status 1
2015-01-20T06:26:49.721626+00:00 heroku[api]: Scale to web=1 by jonathanp.chen@gmail.com
2015-01-20T06:26:49.326889+00:00 heroku[web.1]: State changed from starting to crashed
2015-01-20T06:31:18.624342+00:00 heroku[api]: Release v8 created by jonathanp.chen@gmail.com
2015-01-20T06:31:18.624342+00:00 heroku[api]: Deploy c341409 by jonathanp.chen@gmail.com
2015-01-20T06:31:18.783315+00:00 heroku[web.1]: State changed from crashed to starting
2015-01-20T06:31:23.816864+00:00 heroku[web.1]: Starting process with command `python server.py`
2015-01-20T06:31:26.022804+00:00 app[web.1]: Traceback (most recent call last):
2015-01-20T06:31:26.022829+00:00 app[web.1]:     from Ziggeo import Ziggeo
2015-01-20T06:31:26.022889+00:00 app[web.1]:   File "/app/ZiggeoAuth.py", line 1, in <module>
2015-01-20T06:31:26.022822+00:00 app[web.1]:   File "server.py", line 2, in <module>
2015-01-20T06:31:26.022833+00:00 app[web.1]:   File "/app/Ziggeo.py", line 6, in <module>
2015-01-20T06:31:26.022868+00:00 app[web.1]:     from ZiggeoAuth import ZiggeoAuth
2015-01-20T06:31:26.022914+00:00 app[web.1]:     from Crypto import Random
2015-01-20T06:31:26.022944+00:00 app[web.1]: ImportError: No module named Crypto
2015-01-20T06:31:26.765354+00:00 heroku[web.1]: Process exited with status 1
2015-01-20T06:31:26.780941+00:00 heroku[web.1]: State changed from starting to crashed
2015-01-20T06:31:30.128561+00:00 heroku[api]: Scale to web=1 by jonathanp.chen@gmail.com
2015-01-20T06:32:16.291816+00:00 heroku[api]: Starting process with command `pip freeze` by jonathanp.chen@gmail.com
2015-01-20T06:32:22.391039+00:00 heroku[run.3818]: Awaiting client
2015-01-20T06:32:22.450041+00:00 heroku[run.3818]: Starting process with command `pip freeze`
2015-01-20T06:32:22.864443+00:00 heroku[run.3818]: State changed from starting to up
2015-01-20T06:32:26.170961+00:00 heroku[run.3818]: Process exited with status 0
2015-01-20T06:32:26.175985+00:00 heroku[run.3818]: State changed from up to complete
2015-01-20T06:35:23.336827+00:00 heroku[api]: Starting process with command `pip freeze` by jonathanp.chen@gmail.com
2015-01-20T06:35:28.220089+00:00 heroku[run.2028]: Awaiting client
2015-01-20T06:35:28.249925+00:00 heroku[run.2028]: Starting process with command `pip freeze`
2015-01-20T06:35:28.549991+00:00 heroku[run.2028]: State changed from starting to up
2015-01-20T06:35:31.504519+00:00 heroku[run.2028]: Process exited with status 0
2015-01-20T06:35:31.509078+00:00 heroku[run.2028]: State changed from up to complete
2015-01-20T06:39:26.747374+00:00 heroku[web.1]: State changed from crashed to starting
2015-01-20T06:39:26.641205+00:00 heroku[api]: Deploy 0e2a02a by jonathanp.chen@gmail.com
2015-01-20T06:39:26.641205+00:00 heroku[api]: Release v9 created by jonathanp.chen@gmail.com
2015-01-20T06:39:30.771556+00:00 heroku[web.1]: Starting process with command `python server.py`
2015-01-20T06:39:32.060249+00:00 app[web.1]:   File "/app/Ziggeo.py", line 6, in <module>
2015-01-20T06:39:32.060253+00:00 app[web.1]:     from ZiggeoAuth import ZiggeoAuth
2015-01-20T06:39:32.060221+00:00 app[web.1]: Traceback (most recent call last):
2015-01-20T06:39:32.060322+00:00 app[web.1]: ImportError: No module named Crypto
2015-01-20T06:39:32.060239+00:00 app[web.1]:   File "server.py", line 2, in <module>
2015-01-20T06:39:32.060247+00:00 app[web.1]:     from Ziggeo import Ziggeo
2015-01-20T06:39:32.060278+00:00 app[web.1]:   File "/app/ZiggeoAuth.py", line 1, in <module>
2015-01-20T06:39:32.060297+00:00 app[web.1]:     from Crypto import Random
2015-01-20T06:39:32.763691+00:00 heroku[web.1]: Process exited with status 1
2015-01-20T06:39:32.782855+00:00 heroku[web.1]: State changed from starting to crashed
2015-01-20T06:39:32.783509+00:00 heroku[web.1]: State changed from crashed to starting
2015-01-20T06:39:36.737531+00:00 heroku[web.1]: Starting process with command `python server.py`
2015-01-20T06:39:38.097170+00:00 app[web.1]:   File "/app/Ziggeo.py", line 6, in <module>
2015-01-20T06:39:38.097176+00:00 app[web.1]:     from ZiggeoAuth import ZiggeoAuth
2015-01-20T06:39:38.097208+00:00 app[web.1]:   File "/app/ZiggeoAuth.py", line 1, in <module>
2015-01-20T06:39:38.097242+00:00 app[web.1]:     from Crypto import Random
2015-01-20T06:39:38.097270+00:00 app[web.1]: ImportError: No module named Crypto
2015-01-20T06:39:38.097141+00:00 app[web.1]: Traceback (most recent call last):
2015-01-20T06:39:38.097161+00:00 app[web.1]:   File "server.py", line 2, in <module>
2015-01-20T06:39:38.097169+00:00 app[web.1]:     from Ziggeo import Ziggeo
2015-01-20T06:39:38.778561+00:00 heroku[web.1]: Process exited with status 1
2015-01-20T06:39:38.789902+00:00 heroku[web.1]: State changed from starting to crashed
2015-01-20T06:44:17.047223+00:00 heroku[api]: Deploy f8ac86e by jonathanp.chen@gmail.com
2015-01-20T06:44:17.047223+00:00 heroku[api]: Release v10 created by jonathanp.chen@gmail.com
2015-01-20T06:44:17.250272+00:00 heroku[web.1]: State changed from crashed to starting
2015-01-20T06:44:22.585453+00:00 heroku[web.1]: Starting process with command `python server.py`
2015-01-20T06:44:24.599655+00:00 app[web.1]: Traceback (most recent call last):
2015-01-20T06:44:24.599675+00:00 app[web.1]:   File "server.py", line 2, in <module>
2015-01-20T06:44:24.599682+00:00 app[web.1]:     from Ziggeo import Ziggeo
2015-01-20T06:44:24.599685+00:00 app[web.1]:   File "/app/Ziggeo.py", line 6, in <module>
2015-01-20T06:44:24.599721+00:00 app[web.1]:     from ZiggeoAuth import ZiggeoAuth
2015-01-20T06:44:24.599739+00:00 app[web.1]:   File "/app/ZiggeoAuth.py", line 1, in <module>
2015-01-20T06:44:24.599762+00:00 app[web.1]:     from Crypto import Random
2015-01-20T06:44:24.599814+00:00 app[web.1]: ImportError: No module named Crypto
2015-01-20T06:44:25.309154+00:00 heroku[web.1]: Process exited with status 1
2015-01-20T06:44:25.325736+00:00 heroku[web.1]: State changed from starting to crashed
2015-01-20T06:52:08.560448+00:00 heroku[web.1]: State changed from crashed to starting
2015-01-20T06:52:13.318718+00:00 heroku[web.1]: Starting process with command `python server.py`
2015-01-20T06:52:15.120177+00:00 app[web.1]:   File "/app/ZiggeoAuth.py", line 1, in <module>
2015-01-20T06:52:15.120049+00:00 app[web.1]:   File "server.py", line 2, in <module>
2015-01-20T06:52:15.120056+00:00 app[web.1]:     from Ziggeo import Ziggeo
2015-01-20T06:52:15.120093+00:00 app[web.1]:   File "/app/Ziggeo.py", line 6, in <module>
2015-01-20T06:52:15.120139+00:00 app[web.1]:     from ZiggeoAuth import ZiggeoAuth
2015-01-20T06:52:15.120030+00:00 app[web.1]: Traceback (most recent call last):
2015-01-20T06:52:15.120267+00:00 app[web.1]: ImportError: No module named Crypto
2015-01-20T06:52:15.120207+00:00 app[web.1]:     from Crypto import Random
2015-01-20T06:52:15.911670+00:00 heroku[web.1]: Process exited with status 1
2015-01-20T06:52:15.929388+00:00 heroku[web.1]: State changed from starting to crashed
